static/elements/chromedash-feature.html

```diff
@@ -469,7 +469,7 @@
       if (!link) {
         return;
       }
-      var m = link.match(/[\/|?id=]([0-9]+)$/); // pull out bug number.
+      var m = link.match(/[\/|?id=]([0-9]+)(?:&desc=\d)?$/); // pull out bug number.
```

Comment:
Think we can safely change the entire regex to be: `/id=([0-9]+)/g`. 

That will pull out an id=NUM from anywhere and wouldn't need special handing for other params.